aurelia-slickgrid/src/aurelia-slickgrid/aurelia-slickgrid.ts

```diff
@@ -286,6 +294,14 @@ export class AureliaSlickgridCustomElement {
       }
     }
 
+    // expose GridState Service changes event through dispatch
+    this.gridStateSubscriber = this.ea.subscribe('gridStateService:changed', (gridStateChange: GridStateChange) => {
+      this.elm.dispatchEvent(new CustomEvent(`on-grid-state-service-changed`, {
```

Comment:
it might be good if we always use the `eventPrefix` on dispatched events so we know the events will be unique to the library. That prefix is in the `AureliaSlickgridCustomElement`, but it might be good if we put it in a separate constants file (of if another applicable file exists) so other custom elements can use it too